### PR TITLE
Update document for 1.1.0

### PR DESCRIPTION
**Pull request by gowthamsk-arm**
Signed-off-by: Gowtham Suresh Kumar <gowtham.sureshkumar@arm.com>

### CODE REVIEW

**Review by hug-dev — APPROVED** [review]
Thanks for the update :ok_hand: !

File: src/getting_started/linux_x86.md [hunk 1]
```diff
@@ -150,4 +150,4 @@ TODO
 
 TODO
 
-*Copyright 2021 Contributors to the Parsec project.*
+*Copyright 2022 Contributors to the Parsec project.*
```
[the remark this answers is not in the document]
Comment (gowthamsk-arm) [reply src/getting_started/linux_x86.md:153]: The year is usually updated when the file is modified.:)

**Review by ionut-arm — APPROVED** [review]
👍🏻 Guessing we're merging this after the release is actually tagged?

**Comment by gowthamsk-arm** [comment]
Yes, this will be open until we create a release tag. :)

**Review by mohamedasaker-arm — APPROVED** [review]

**Pull request merged**